smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/lang/ReservedWords.kt

```diff
@@ -83,6 +83,7 @@ val reservedWords =
         "operator",
         "optional",
         "override",
+        "package",
```

Comment:
Adding `package` as a Swift reserved word so that it gets escaped with backticks in generated code.

There actually do seem to be services that already use `package` as an identifier without escaping it, but to date it seems not to cause any problems.